static/sass/_pattern_lists.scss

```diff
@@ -312,7 +312,8 @@
 }
 
 @mixin ubuntu-p-list-ticked {
-  .p-list--ticked {
+  .p-list--ticked,
+  .p-list--ticked--small {
```

Comment:
Is this a new component? Can you raise an issue on Vanilla and comment it here please?